ghost/content.md

```diff
@@ -1,6 +1,8 @@
 # Ghost
 
-Ghost is a free and open source blogging platform written in JavaScript and distributed under the MIT License, designed to simplify the process of online publishing for individual bloggers as well as online publications.
+Ghost is an independent platform for publishing online by web and email newsletter. It has user signups, gated access and subscription payments built-in (with Stripe) to allow you to build a direct relationship with your audience. It's fast, user-friendly, and runs on Node.js & MySQL8.
+
+Ghost is used in production by Apple, OpenAI, TheAtlantic, SkyNews, DuckDuckGo, Buffer, and tens of thousands of companies, news websites and independent creators. Learn more about the project on [Ghost.org](https://ghost.org)
 
 > [wikipedia.org/wiki/Ghost_(blogging_platform)](http://en.wikipedia.org/wiki/Ghost_%28blogging_platform%29)
```

Comment:
This was intended as a "source"/citation for the previous blurb, so if you want to keep the link, it might be worth working into the prose somewhere (or giving a sentence by itself) but otherwise should probably be removed. :eyes:

Reply:
I understand the intent, but per your previous comment about bitrot - Wikipedia is extremely out of date and is generally not a great source up-to-date information about open source projects!

Would you be open to us updating the link to the official project website/docs maintained by the core team and the non-profit foundation? Ghost.org

Reply:
Yeah, but that's the link that's already at the end of your previous paragraph, so I still think this line should probably just be removed. :sweat_smile:

Reply:
👍 updated